tests/helper/tests/packages_musthave.py

```diff
@@ -54,8 +54,9 @@ def packages_musthave(client, testconfig):
         # * the architecture as a variable in the package name
         elif package.endswith(r"-${arch}"):
             package = package.replace(r"${arch}", arch)
+
         # explicitly excluded packages are allowed to miss 
-        elif package in exclude:
+        if package in exclude:
             continue
```

Comment:
I'm not quite sure if we should print an info or warning for the pytest output (e.g. `logger.info(f"Package {package} is excluded. Continuing")`). However, lgtm